frontend/src/features/films/Film.js

```diff
@@ -18,35 +18,45 @@ const Film = () => {
   const [tickets, setTickets] = useState({
     id: "",
     name: "",
-    price: 0,
+    ticket_price: 0,
     amount: 1,
     time: "13:00",
   });
+  console.log(tickets);
 
   useEffect(() => {
     const selectedFilm = { film_name: params.film_name };
     dispatch(fetchTickets(selectedFilm));
   }, [dispatch, params]);
 
+  useEffect(() => {
```

Comment:
What is this doing?

Reply:
this is for setting the initial state for the ticket